app/DoctrineMigrations/Version20190705153145.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190705153145 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('CREATE TABLE restaurant_pledge_vote (id SERIAL NOT NULL, user_id INT NOT NULL, pledge_id INT DEFAULT NULL, voted_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE UNIQUE INDEX UNIQ_9A6FB9B1A76ED395 ON restaurant_pledge_vote (user_id)');
```

Comment:
Problem here is that the generated SQL contains `CREATE UNIQUE INDEX` on `user_id`.
It means users can vote once. We want them to vote once **per pledge**. 
Probably the mapping is wrong in `Restaurant.PledgeVote.orm.yml`. Maybe the association mapping between `PledgeVote` & `ApiUser` should be `ManyToMany`?